gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -577,6 +966,82 @@ def main(args):  # noqa: D103
             )
 
             ht.describe()
+        if args.prepare_vcf_header:
+            logger.info("Preparing VCF header dict...")
+            res = resources.prepare_vcf_header
+            res.check_resource_existence()
+            ht = res.release_ht.ht()
+            validated_ht = res.validated_ht.ht()
+
+            header_dict = prepare_vcf_header_dict(
+                ht,
+                validated_ht,
+                bin_edges=make_hist_bin_edges_expr(
+                    ht,
+                    include_age_hists=True,
+                ),
+                age_hist_data=ht.age_distribution,
```

Comment:
I think it needs to be `ht.histograms.age_distribution` here because the struct changed?

Reply:
Maybe it also explains why it says so: 
```commandline
WARNING (gnomad.assessment.validity_checks 841): age_hist_het_bin_edges in info field does not exist in VCF header!
WARNING (gnomad.assessment.validity_checks 841): age_hist_hom_bin_edges in info field does not exist in VCF header!
WARNING (gnomad.assessment.validity_checks 841): SB in info field does not exist in VCF header!
```

Reply:
I will need more time for the "SB" thing.

Reply:
The SB is just not in the info dictionary, its in format. I just need to confirm the definition as noted in the notebook

Reply:
The age_distribution is a global, it is not within the histogram struct